execution/gethexec/node.go

```diff
@@ -185,7 +185,20 @@ func CreateExecutionNode(
 		if err != nil {
 			return nil, err
 		}
-		txPublisher = sequencer
+		if config.Sequencer.Espresso {
+			espressoSequencer, err := NewEspressoSequencer(execEngine, seqConfigFetcher)
+			if err != nil {
+				return nil, err
+			}
+			switchSequencer, err := NewSwitchSequencer(sequencer, espressoSequencer, parentChainReader.Client(), seqConfigFetcher)
```

Comment:
Enable the automated escape hatch to restore liveness during HotShot liveness failures.